// Copyright 2015 gRPC authors.
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//     http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.

syntax = "proto3";

option java_multiple_files = true;
package com.oppo.ohacker.scheduler;

// The Scheduler service definition.
service SchedulerService {
    rpc ready (ReadyRequest) returns (ReadyResponse) {
    }
}

// The request message
message ReadyRequest {
    string playerHost = 1;
    int32 playerPort = 2;
    int32 matchId = 3;
    int32 teamId = 4;
}

// The response message:ok
message ReadyResponse {
    string msg = 1;
}
